views/todo/create2.ejs

```diff
@@ -0,0 +1,24 @@
+<div>
+    <h2>Create Todo</h2>
+
+    <form method="POST" action="/user/create">
+        <div class="form-row mb-3">
+            <input
+                placeholder="Title of Todo"
+                class="form-control" name="title"
+                />
+        </div>
+        <div class="form-row mb-3">
+            <input
+                    placeholder="Todo Description"
+                    class="form-control" name="description"
+            />
+        </div>
+
+    </form>
+
+    <form method="POST" action="/user/create?_method=PUT" />
```

Comment:
you shdnt have a separate PUT here ! the create todo item should be a single put request

Reply:
The changing of the isComplete status will be a separate button in the main.ejs -> (which will be a put request there)